Content.Shared/_RMC14/Xenonids/Construction/Tunnel/TunnelRenamerComponent.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Content.Shared._RMC14.Xenonids.Construction.Tunnel;
+[RegisterComponent]
```

Comment:
This needs to be a networked component as well or the client will never know about it